{"name": "test_first", "status": "failed", "statusDetails": {"message": "AssertionError: assert 10 == 20", "trace": "def test_first():\n        a = 10\n        b = 20\n    \n>       assert a == b\nE       assert 10 == 20\n\nSeleniumProgram\\test_first_program.py:5: AssertionError"}, "start": 1672762896392, "stop": 1672762896393, "uuid": "35374b68-c020-4f65-93ba-f0e1dd2919ed", "historyId": "54701c13c5f1bf9e48a35aaad587bd9f", "testCaseId": "bc64b7aa61bef38ed465753d8f157431", "fullName": "test_first_program#test_first", "labels": [{"name": "suite", "value": "test_first_program"}, {"name": "host", "value": "HP"}, {"name": "thread", "value": "24940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_first_program"}]}